packages/nimble-components/src/table/tests/table-sorting.spec.ts

```diff
@@ -733,4 +733,165 @@ describe('Table sorting', () => {
             expect(getRenderedRecordIds()).toEqual(['3', '4', '1', '2']);
         });
     });
+
+    describe('interactive sorting', () => {
```

Comment:
I think we're missing test coverage of `sorting-disabled`.

Reply:
Added